### ISSUE TITLE
Make images accessible only by root

### ISSUE THREAD

**Issue by fottsia** [ltsp/ltsp#336]
Hi, currently LTSP images are world-readable:

```shell
# ls -la /srv/ltsp/images
drwxr--r-- 2 root root       4096 Sep  7 11:18 .
-rw-r--r-- 1 root root 2601562112 Sep  7 11:18 x86_64.img
-rw-r--r-- 1 root root 2601582592 Aug 21 11:11 x86_64.img.old
```

Usually there are no secrets inside the images, and NFSv3 isn't a secure protocol, but it would be better to make them accessible only by root anyway, e.g. in case wireguard or NFSv4 are used.

**Comment by alkisg** [comment]
Fixed in #eb362d2, please test that it doesn't break anything. To make it easier for the sysadmin to revert this change, I only set 0700 mode for the images directory, not for the files. So to revert to the previous behavior, one would run:

```shell
chmod 755 /srv/ltsp/images
```

**Comment by fottsia** [comment]
It seems to work fine here.
